### DISCUSSION TITLE
Deploying with trellis?

### DISCUSSION THREAD

**Discussion by Guillaume-Tnk** [understrap/understrap-child#286] [Q&A] [answered]
Anyone using trellis https://roots.io/trellis/ and bedrock https://roots.io/bedrock/ with understrap?

I'm having a hard time getting my CSS to be applied in production?

Would love to get guidance on this...

**Comment by danemorgan** [comment]
I wouldn't expect your CSS issues to be related to the local environment you choose. I've never used these tools, but things should proceed as normal if they provide a basic LAMP stack and WordPress installation.

Could you elaborate on what, specifically, is happening with your CS?

**Reply by Guillaume-Tnk** [reply]
@danemorgan I'm just stupid, it was the browser cache (ㆆ_ㆆ) It's all working flawlessly.

**Reply by danemorgan** [reply]
Haha. I believe we've all been in that exact spot a time or three.

**Reply by 0dp** [reply]
...a day

**Reply by Guillaume-Tnk** [reply]
Not that this topic got massive interest but for those interested, I just made a step by step on how to develop and deploy understrap with Trellis. I'm not a specialist, far from it, but I think it's awesome and if that can help anyone, [here](https://docs.google.com/document/d/1d3aAGTp4e75q3xfcUHUB08YvHxv-OMRzxfRCol98uCc/edit?usp=sharing) it is.

Feel free to let me know if something isn't clear so I update it.

Cheers !!

**Comment by bacoords** [comment, marked as answer]
Copying from @Guillaume-Tnk so we can mark his work as the answer: 

> Not that this topic got massive interest but for those interested, I just made a step by step on how to develop and deploy understrap with Trellis. I'm not a specialist, far from it, but I think it's awesome and if that can help anyone, [here](https://docs.google.com/document/d/1d3aAGTp4e75q3xfcUHUB08YvHxv-OMRzxfRCol98uCc/edit?usp=sharing) it is.
> 
> Feel free to let me know if something isn't clear so I update it.
> 
> Cheers !!
